trento/xml/article_sap_trento.xml

```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           </step>
         </procedure>
       </section>
+
+    <section>
+      <title>Poda descriptiona and logs</title>
+      <para> In additon to the output of the Trento Support Plugin and the Dump Scenario script, the descriptions and logs of the
+       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
+       easily retrieved with the <command>kubectl</command> command, for which you need a host where command <command>kubectl</command>
+       is installed and connected to the K8s cluster where Trento Server is running.
```

Comment:
* I would split the sentence
* Use `&t.server;` entity
* "easily": would drop that. Easy/difficult categories are quite subjective and we don't want that our readers feel dump if they perceive something as not so easy.
* The last sentence tries to put too much information into one sentence.

If I take all this into considerations, I'll end up with this:

```suggestion
       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
       retrieved with the <command>kubectl</command> command. You need a host where the <command>kubectl</command> command
       is installed and connected to the K8s cluster. The K8s cluster has to run the &t.server;.
```

Reply:
I'm fine with splitting the last sentence in two parts, but maybe three is to much. I would go for:

These descriptions and logs can be retrieved with the <command>kubectl</command> command. You need a host where the <command>kubectl</command> command is installed and connected to the K8s cluster where &t.server; is running.

After all, the sentence "You need a host where the <command>kubectl</command> command is installed and connected to the K8s cluster where &t.server; is running." is already used in the conditions to use the Trento Support Plugin and the Dump Scenario scrip.